Fix Workflow plug-in version and Building from source section (#148)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/tooling/kn-plugin-workflow-overview.adoc

--

[[proc-build-sw-project-source]]
== Building from source

> As an alternative, you can build the Workflow plug-in from source.

Reviewer: ```suggestion
As an alternative, you can build the workflow plug-in from the source.
```

----
--

. Navigate to the cloned `kie-tools` directory and enter the following command to install the dependencies and link the repository packages:
+
--
.Navigate to the repository, install dependencies, and link packages
[source,shell]
----
cd kie-tools
pnpm bootstrap
----
--

> . In the repository root, enter the following command to build the Workflow plug-in:

Reviewer: ```suggestion
. In the repository root, enter the following command to build the workflow plug-in:
```

+
--

> .Build Workflow plug-in from source

Reviewer: ```suggestion
.Build workflow plug-in from source
```

[source,shell]
----
pnpm -r -F kn-plugin-workflow... build:dev
----
--

> You have successfully built the Workflow plug-in project from source. This _Procedure_ generates an artifact compatible with your OS architecture in the `kie-tools/packages/kn-plugin-workflow/dist` folder.

Reviewer: ```suggestion
You have successfully built the workflow plug-in project from the source. This procedure generates an artifact compatible with your operating system architecture in the `kie-tools/packages/kn-plugin-workflow/dist` folder.
```
